src/solvers/smtlib/smtlib_conv.cpp

```diff
@@ -104,12 +104,38 @@ smt_convt *create_new_smtlib_solver(
   array_iface **array_api,
   fp_convt **fp_api)
 {
+  if(!options.get_bool_option("smt-formula-only"))
+  {
+    log_error(
+      "[smtlib] the smtlib interface does not support solving for now. Please, "
+      "use it with --smt-formula-only");
+    abort();
```

Comment:
Would it be possible to make this a warning instead? The `--smtlib-solver-prog` option still works even though parsing the replies from the external program is somewhat buggy. Making this a warning would help in debugging it.

Reply:
Sure